packages/mc-scripts/src/config/optimizations.ts

```diff
@@ -0,0 +1,66 @@
+import chalk from 'chalk';
+
+// Dependencies to be split/grouped into separate chunks.
+// This is useful to reduce the main bundle size and have more
+// dedicated caching strategy for specific chunks.
+const manualChunks = {
+  'commercetools-uikit-icons': ['@commercetools-uikit/icons'],
+  moment: ['moment', 'moment-timezone'],
+  'app-shell': [
+    '@commercetools-frontend/application-shell',
+    '@commercetools-frontend/application-shell-connectors',
+  ],
+};
+
+const removeNonExistantDependencies = (
+  manualChunks: Record<string, string[]>,
+  appDependencies: string[]
+) => {
+  return Object.entries(manualChunks).reduce((acc, [chunkName, vendors]) => {
+    const existingVendors = vendors.filter((vendor) =>
+      appDependencies.includes(vendor)
+    );
+    if (existingVendors.length > 0) {
```

Comment:
```suggestion
    const hasExistingVendors = vendors.some((vendor) =>
      appDependencies.includes(vendor)
    );
    if (hasExistingVendors) {
```

Reply:
Let's picture this scenario:
```ts
const manualChunks = {
  'chunk-A': ['dep-a', 'dep-b'];
}
```

With the current code, if the app does not have 'dep-b' anymore, the manual chunk would still be created with `dep-a`. This would not work if we apply your suggestion.

Reply:
Thanks, nevermind then. I misread the code 🙈